wordpress/wp-content/plugins/memberful-wp/src/acl.php

```diff
@@ -195,7 +214,135 @@ function memberful_wp_extract_slug_ids_and_user($args) {
  * @param integer $post_id ID of the post that should have access checked
  */
 function memberful_can_user_access_post( $user, $post ) {
```

Comment:
```suggestion
function memberful_can_user_access_post( $user_id, $post_id ) {
```

It looks that both arguments are IDs so we should rename that. Otherwise `"user-{$user}--post-{$post}"` from below gets pretty confusing. At first I thought that we are using objects in cache keys.

Reply:
This is actually not related to this PR so we can do it later.